**Test B Solutions (MATH 2511)**

(1) Evaluate the following limit. $$\lim_{x\rightarrow\frac{\pi}{2}}\frac{2x-\pi}{\cos{}x}$$

$$\lim_{x\rightarrow\frac{\pi}{2}}\frac{2x-\pi}{\cos{}x}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow\frac{\pi}{2}}\frac{2}{-\sin{}x}=\frac{2}{-\sin\frac{\pi}{2}}=-\frac{2}{1}=-2$$

Evaluate the following limit. $$\lim_{x\rightarrow{}0}\frac{\sin{}t^{2}}{t}$$

$$\lim_{x\rightarrow{}0}\frac{\sin{}t^{2}}{t}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow{}0}\frac{2t\cos{}t^{2}}{1}=\frac{1\cdot{}0}{1}=0$$

(2) Simplify, using the definition of hyperbolic trigonometric functions,$$\sqrt{2\pi\sinh\left(\ln\pi\right)+1}$$

$$\sqrt{2\pi\sinh\left(\ln\pi\right)+1}=\sqrt{2\pi\left(\frac{1}{2}\left(e^{\ln\pi}-e^{-\ln\pi}\right)\right)+1}=\sqrt{\pi\left(\pi-\frac{1}{\pi}\right)+1}=\sqrt{\pi^{2}}=\pi$$

(3) Analyze the following function (do not worry about asymptotes and whether the function is even/odd). $$f(x)=\frac{2x-1}{2x^{2}}$$

<img src="ttbana.png">

domain: $\{x\in\mathbb{R}|x\neq{}0\}=\mathbb{R}\setminus\{0\}$

$x$-intercepts: $\left\{\frac{1}{2}\right\}$, the solution set of $2x-1=0$

$$f'(x)=\frac{-x+1}{x^{3}}$$

critical points: $\left\{1\right\}$, the solution set of $-x+1=0$

$$f''(x)=\frac{2x-3}{x^{4}}$$

inflection points: $\left\{\frac{3}{2}\right\}$, the solution set of $2x-3=0$

the critical point is a maximum because $f''(1)=-1$


range: $\{x\in\mathbb{R}|x\leq\frac{1}{2}\}$

Analyze the following function (do not worry about asymptotes and whether the function is even/odd). $$f(x)=\frac{x-1}{3x^{2}}$$

domain: $\{x\in\mathbb{R}|x\neq{}0\}=\mathbb{R}\setminus\{0\}$

$x$-intercepts: $\left\{1\right\}$, the solution set of $x-1=0$

$$f'(x)=\frac{-x+2}{3x^{3}}$$

critical points: $\left\{2\right\}$, the solution set of $-x+2=0$

$$f''(x)=\frac{2x-6}{3x^{4}}$$

inflection points: $\left\{3\right\}$, the solution set of $2x-6=0$

the critical point is a maximum because $f''(2)=-\frac{1}{24}$

range: $\{x\in\mathbb{R}|x\leq\frac{1}{12}\}$

(4) Differentiate $$g(t)=\ln\left(\sinh\left(2t\right)\right)$$

$$g'(t)=2\coth(2t)$$

Differentiate $$g(t)=\ln\left(\cosh\left(2t\right)\right)$$

$$g'(t)=2\tanh(2t)$$

(5) An open-top box is to be made by cutting small congruent squares from the corners of a 13 inch by 13 inch sheet of tin and bending up the sides. How large should the squares cut from the corners be to make the box hold as much as possible?

<img src="gc-termtestB-v1-01.png">

$$V(x)=(13-2x)^{2}x$$

$$V'(x)=12x^{2}-104x+169$$

In [1]:
solve(12*x^2-104*x+169,x)

[x == (13/6), x == (13/2)]

In [2]:
n(13/6)

2.16666666666667

Answer: The squares cut from the corners to make the box hold as much as possible should have $x=13/6$ as their side.

An open-top box is to be made by cutting small congruent squares from the corners of a 11 inch by 11 inch sheet of tin and bending up the sides. How large should the squares cut from the corners be to make the box hold as much as possible?

$$V(x)=(11-2x)^{2}x$$

$$V'(x)=12x^{2}-88x+121$$

In [2]:
solve(12*x^2-88*x+121,x)

[x == (11/6), x == (11/2)]

In [3]:
n(11/6)

1.83333333333333

Answer: The squares cut from the corners to make the box hold as much as possible should have $x=11/6$ as their side.

(6) Evaluate the following limit. $$\lim_{x\rightarrow\frac{\pi}{4}}\frac{\sin{}x-\cos{}x}{x-\frac{\pi}{4}}$$

$$\lim_{x\rightarrow\frac{\pi}{4}}\frac{\sin{}x-\cos{}x}{x-\frac{\pi}{4}}\stackrel{\mbox{LHR}}{=}\lim_{x\rightarrow\frac{\pi}{4}}\frac{\cos{}x+\sin{}x}{1}=\frac{2}{\sqrt{2}}=\sqrt{2}$$

(7) To calculate a planet's space coordinates, we have to solve equations like $$x=1+0.4\sin{}x$$ Graphing the function $f(x)=x-1-0.4\sin{}x$ suggests that the function has a root near $x=1.4$. Use one application of Newton's method to improve this estimate. That is, start with $x_{1}=1.4$ and find $x_{2}$.

$$f'(x)=1-0.4\cos{}x$$

In [14]:
n(1.4-((1.4-1-0.4*sin(1.4))/(1-0.4*cos(1.4))))

1.39375533698282

To calculate a planet's space coordinates, we have to solve equations like $$x=1+0.7\sin{}x$$ Graphing the function $f(x)=x-1-0.7\sin{}x$ suggests that the function has a root near $x=1.7$. Use one application of Newton's method to improve this estimate. That is, start with $x_{1}=1.7$ and find $x_{2}$.

$$f'(x)=1-0.7\cos{}x$$

In [15]:
n(1.7-((1.7-1-0.7*sin(1.7))/(1-0.7*cos(1.7))))

1.69464806451177

(8) A conical tent is to have $1000$ft$^{3}$ capacity. Find its dimensions if the amount of canvas is to be a minimum. Disregard the floor. Note that the formula for the volume of a cone is $$V=\frac{1}{3}r^{2}\pi{}h$$ The formula for the area of the lateral surface of a cone (without the floor) is $$A=\pi{}rs$$ where $s$ is the slant height (see diagram).


<img src="coneslantheight.png">

**Step 1:** $A(r)=\pi{}rs$

**Step 2:** $s=\sqrt{r^{2}+h^{2}}$ and $\frac{1}{3}r^{2}\pi{}h=1000$ therefore $$s=\sqrt{r^{2}+\frac{9\cdot{}10^{6}}{r^{4}\pi^{2}}}$$ $$A(r)=\pi{}r\sqrt{r^{2}+\frac{9\cdot{}10^{6}}{r^{4}\pi^{2}}}=\pi{}\sqrt{r^{4}+\frac{9\cdot{}10^{6}}{r^{2}\pi^{2}}}$$

**Step 3:** $$A'(r)=\pi\cdot\frac{4r^{3}-\frac{18\cdot{}10^{6}}{r^{3}\pi^{2}}}{2\sqrt{r^{4}+\frac{9\cdot{}10^{6}}{r^{2}\pi^{2}}}}$$

It follows that, solving $A'(r)=0$, $$4r^{3}=\frac{18\cdot{}10^{6}}{r^{3}\pi^{2}}$$ Isolate $r$ for $$r=10\cdot\left(\frac{9}{2\pi^{2}}\right)^{\frac{1}{6}}$$

In [11]:
r=((9/(2*pi^2))^(1/6))*10

Since $h=\frac{3000}{r^{2}\pi}$

In [12]:
h=3000/(r^2*pi)

In [13]:
(n(r),n(h))

(8.77308077654739, 12.4070098179880)

Answer: if the amount of the canvas is to be at a minimum, then the dimensions should be approximately $r=8.77$ feet and $h=12.41$ feet.